ENIGMAsystem/SHELL/Universal_System/Resources/loading.cpp

```diff
@@ -123,6 +123,11 @@ namespace enigma
     //Initialize extensions
     enigma::extensions_initialize();
 
+    // apply global game settings
+    enigma_user::window_set_sizeable(isSizeable);
```

Comment:
Do any of these routines depend on room dimensions?

Reply:
No, none of them depend on the room dimensions.

Reply:
Please actually test this on Windows and Linux robert there is a real reason I did this, and that reason you'll find out when you test it.